src/types/xToken.d.ts

```diff
@@ -59,6 +74,21 @@ export type IContracts =
   | typeof TRADE_ACCOUNTING
   | typeof UNISWAP_V2_PAIR
   | typeof XTK_MANAGEMENT_STAKING_MODULE
+  | typeof LENDING_COMPTROLLER
```

Comment:
Create a separate type for `ILendingContracts` and expand the type of `IContracts` using that. Similar pattern to `ITokenSymbols` or `IXU3LPContracts`